readme.md

```diff
@@ -25,7 +25,7 @@ Your Answer:
 Describe at a high level how we use jQuery to submit a form via AJAX.
```

Comment:
Hi,

My AJAX is pretty weak. I am going to try the giphy option from the Thanksgiving homework. After I do that I will get back to you with questions by Thursday at the latest... Also old stuff, I have a copy of choose your own adventure that I can turn in Wednesday AM.
> On Dec 15, 2015, at 2:45 PM, Adrian Maseda <notifications@github.com> wrote:
> 
> In readme.md <https://github.com/ga-dc/wdi7-quiz9/pull/34#discussion_r47687245>:
> 
> > @@ -25,7 +25,7 @@ Your Answer:
> >  Describe at a high level how we use jQuery to submit a form via AJAX.
> Noticed you left Question 1 blank. How are you feeling about AJAX? Included an answer for the question below. Happy to sit down and talk about it. This is a key JS concept that you'll want to be familiar with.
> 
> $.ajax({
>   url: "http://kittengifs.com/gifs"
>   method: "get"
> })
>   .then(function(results){
>     console.log(results.length)
>   });
> —
> Reply to this email directly or view it on GitHub <https://github.com/ga-dc/wdi7-quiz9/pull/34/files#r47687245>.
>